plugins/outputs/riemann/riemann.go

```diff
@@ -24,8 +26,12 @@ var sampleConfig = `
   url = "localhost:5555"
   ## transport protocol to use either tcp or udp
   transport = "tcp"
-  ## separator to use between input name and field name in Riemann service name
+  ## separator to use between measurement name and field name in Riemann service name
   separator = " "
+  ## set measurement name as a Riemann tag instead of prepending it to the Riemann service name
+  measurement_as_tag = false
+  ## list of Riemann tags, if specified use these instead of any Telegraf tags
+  tags = ["telegraf","custom_tag"]
```

Comment:
I think that that may make sense for your use-case, but the vast majority of telegraf users are collecting metrics from a diverse array of sources, not just a single type of source (postgres_cluster) that they're going to want to hardcode into their config file.

What if the riemann output allowed selecting particular telegraf tags to have their values applied as riemann tags? so your config could look something like this?:

```
[global_tags]
  deployment = "postgresql-dev-2"
  id = "1e612b44-e92f-4d27-9f30-5e2f53947870"
  az = "z1"
  description = "postgres_cluster"

[[outputs.riemann]]
  url = "localhost:5555"
  measurement_as_tag = true
  rieman_tag_keys = ["description"]

...

#riemann.codec.Event{
:host "postgresql-1e612b44-e92f-4d27-9f30-5e2f53947870", :state nil, :description nil, :ttl nil,
:measurement "disk", :service "used_percent", :metric 73.16736001949994, :path "/boot", :fstype "ext4",
:tags ["telegraf" "postgres_cluster"],
:deployment "postgresql-dev-2", :id "1e612b44-e92f-4d27-9f30-5e2f53947870", :az "z1", :time 1475605021}
```